src/pytorch_lightning/core/module.py

```diff
@@ -1875,12 +1875,16 @@ def to_onnx(self, file_path: Union[str, Path], input_sample: Optional[Any] = Non
             ...     def forward(self, x):
             ...         return torch.relu(self.l1(x.view(x.size(0), -1)))
 
-            >>> import os, tempfile
+            >>> model = SimpleModel()
+
+            >>> import os, tempfile, doctest
+            >>> doctest.ELLIPSIS_MARKER = '-etc-'
```

Comment:
This is needed as the older version do not print anything, and the new version print blank line...